PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -3922,6 +3923,12 @@ void FindReplaceDlg::execSavedCommand(int cmd, uptr_t intValue, const generic_st
 						break;
 					}
 
+					case IDC_COPY_MARKED_TEXT:

```

Comment:
If https://github.com/notepad-plus-plus/notepad-plus-plus/pull/13407/commits/8f4a1850a1f9d18c434715c662661eafa21d8384#r1148360378 is done, this change could probably be eliminated.